README.md

```diff
@@ -2,28 +2,363 @@
+    ...
+    ...
+
+    var name: String
+
+    var value: AnyObject
+}
+```
+
+```Swift
+extension Sequence where Iterator.Element == Attribute {
+
+    /// Creates dictionary from sequence of attributes by merging them together. String is name of case and AnyObject value corresponding to it.
+    func toDictionary() -> [String: AnyObject]
+}
+```
+
+#### String
+
+```Swift
+extension String {
+
+    func attributed(_ attributes: [Attribute]) -> NSAttributedString
+
+    func attributed(_ attributes: Attribute...) -> NSAttributedString
+}
+```
+
+### Percent
+
+```Swift
+/// Returns input / 100.
+postfix func %(input: CGFloat) -> CGFloat
+```
+
+### UIButton
+
+```Swift
+extension UIButton {
+
+    init(title: String)
+
+    func setBackgroundColor(_ color: UIColor, forState state: UIControlState)
+}
+```
+
+### UICollectionView
+
+```Swift
+extension UICollectionView {
+
+    init(collectionViewLayout layout: UICollectionViewLayout)
+}
+```
+
+### UIColor
+
+```Swift
+extension UIColor {
+
+    /// Accepted formats: "#RRGGBB" and "#RRGGBBAA".
+    init(hex: String)
+
+    init(rgb: UInt)
+
+    init(rgba: UInt)
+
+    /// Increases color's brightness.
+    func lighter(by percent: CGFloat) -> UIColor
+
+    /// Reduces color's brightness.
+    func darker(by percent: CGFloat) -> UIColor
+
+    /// Increases color's saturation.
+    func saturated(by percent: CGFloat) -> UIColor
+
+    /// Reduces color's saturation.
+    func desaturated(by percent: CGFloat) -> UIColor
+
+    /// Increases color's alpha.
+    func fadedIn(by percent: CGFloat) -> UIColor
+
+    /// Reduces color's alpha.
+    func fadedOut(by percent: CGFloat) -> UIColor
+}
+```
+
+### UIEdgeInsets
+
+```Swift
+extension UIEdgeInsets {
+
+    init(top: CGFloat, left: CGFloat, bottom: CGFloat, right: CGFloat)
+
+    init(_ all: CGFloat)
+
+    init(horizontal: CGFloat, vertical: CGFloat)
+
+    init(horizontal: CGFloat, top: CGFloat, bottom: CGFloat)
+
+    init(vertical: CGFloat, left: CGFloat, right: CGFloat)
+}
+```
+
+### UIFont
+
+```Swift
+extension UIFont {
+
+    init(_ name: String, _ size: CGFloat)
+}
+```
+
+### UILabel
+
+```Swift
+extension UILabel {
+
+    init(text: String)
+}
+```
+
+### UIOffset
+
+```Swift
+extension UIOffset {
+
+    init(_ all: CGFloat)
+
+    init(horizontal: CGFloat)
+
+    init(vertical: CGFloat)
+}
+```
+
+### UITableView
+
+```Swift
+extension UITableView {
+
+    init(style: UITableViewStyle)
+}
+```
+
+## Versioning
+
+This library uses semantic versioning. Until version 1.0 API breaking changes may occur even in minor versions. We consider version 0.5 to be prerelease, which means that API should be stable but is not tested yet in real project. After that testing we make needed adjustmens and bump the version to 1.0 (first release).
+
 ## Author
 
-Tadeas Kriz, tadeas@brightify.org
+* Tadeas Kriz, [tadeas@brightify.org](mailto:tadeas@brightify.org)
+* Filip Dolník, [filip@brightify.org](mailto:filip@brightify.org)
+
+## Used libraries
```

Comment:
I'd rather say `## Used libraries in tests` to not imply these are runtime dependencies.